cmd/podman/sign.go

```diff
@@ -138,7 +138,7 @@ func signCmd(c *cli.Context) error {
 			return errors.Wrapf(err, "error creating new signature")
 		}
 
-		sigStoreDir = fmt.Sprintf("%s/%s", sigStoreDir, strings.Replace(repos[0][strings.Index(repos[0], "/")+1:len(repos[0])], ":", "=", 1))
+		sigStoreDir = filepath.Join(sigStoreDir, strings.Replace(repos[0][strings.Index(repos[0], "/")+1:len(repos[0])], ":", "=", 1))
```

Comment:
My brain really cannot decrypt the `strings.Replace(...)` call (non-blocking).

Reply:
@QiWang19 Please explain what this is doing.

Reply:
It is just the folder name to store the signature, according to atomic it is ```/var/lib/atomic/sigstore/library/alpine@sha256=621c2f39f8133acb8e64023a94dbdf0d5ca81896102b9e57c0dc184cadaf5528```,
so I replace the ```:``` in the digest with ```=```

Reply:
The strange part of this is the multi-level array indexed by a strings.Index on the first level... Is that a slice index, trying to trim repos[0]?

Reply:
@mheon , yes...

Reply:
@QiWang19 Break this into multi lines with variable names that tell us what is going on.  6 months from now you will have a hard time deciphering this.